homework/index.js

```diff
@@ -30,17 +35,125 @@
     return elem;
   }
 
+  function createDetails(leftColumn, repositories) {
```

Comment:
This function receives a single repository object. Therefore the second argument should be `repository`. The name `repositories` (plural) implies an array.

Reply:
fixed 👍